test-cypress/integration/monitor/monitor.queries.spec.js

```diff
@@ -0,0 +1,24 @@
+describe('Monitor Queries', () => {
+
+    let repositoryId;
+
+    before(() => {
+        repositoryId = 'monitoring-repo' + Date.now();
+        cy.createRepository({id: repositoryId});
+    });
+
+    beforeEach(() => {
+        cy.presetRepositoryCookie(repositoryId);
+
+        cy.visit('/monitor/queries');
+    });
+
+    after(() => {
+        cy.deleteRepository(repositoryId);
+    });
+
+    it('Initial state ', () => {
+        cy.get('.no-running-queries-alert').contains('No running queries or updates.');
```

Comment:
Maybe also `.should('be.visible')` ? Cause `contains` will check for existence only, I think.